h/migrations/versions/9e6b4f70f588_removing_trailing_from_pdf_urns.py

```diff
@@ -0,0 +1,141 @@
+"""Remove trailing # from PDF URNs.
+
+Revision ID: 9e6b4f70f588
+Revises: 467ea2898660
+Create Date: 2016-06-21 17:50:14.261947
+
+"""
+# pylint: disable=invalid-name, wrong-import-position
+from __future__ import unicode_literals
+
+import logging
+
+# revision identifiers, used by Alembic.
+revision = '9e6b4f70f588'
+down_revision = 'ccebe818f8e0'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+
+from h.api.db import types
+
+Base = declarative_base()
+Session = sessionmaker()
+
+
+log = logging.getLogger(__name__)
+
+
+class Annotation(Base):
+    __tablename__ = 'annotation'
+    id = sa.Column(types.URLSafeUUID,
+                   server_default=sa.func.uuid_generate_v1mc(),
+                   primary_key=True)
+    target_uri = sa.Column('target_uri', sa.UnicodeText)
+
+
+class DocumentURI(Base):
+    __tablename__ = 'document_uri'
+    id = sa.Column(sa.Integer, autoincrement=True, primary_key=True)
+    claimant = sa.Column('claimant', sa.UnicodeText, nullable=False)
+    claimant_normalized = sa.Column('claimant_normalized',
+                                    sa.UnicodeText,
+                                    nullable=False)
+    uri = sa.Column('uri', sa.UnicodeText, nullable=False)
+    uri_normalized = sa.Column('uri_normalized',
+                               sa.UnicodeText,
```

Comment:
It doesn't matter that much but two things to note:

1. There's no need to specify the column name twice when using `declarative_base`: `uri = sa.Column(sa.UnicodeText, )` will work fine.
2. Usually in migrations you don't need to specify all the column properties, just the types, so sqla knows how to map resultsets onto these objects.